README.md

```diff
@@ -146,4 +139,10 @@ Example to fetch at most two SettledDvPs and check if one of them has a payment
 If the boolean argument (`exact`) is true, each incoming contract must match at least one predicate.
 Otherwise, it may skip several events until all predicates are matched.
 
+### Running tests
+
+The library uses JUnit4 to provide its functionalities. Several build systems (e.g. Maven, Sbt) support running JUnit tests easily.
+In Maven, one can use either *maven-failsafe-plugin* or *maven-surefire-plugin* to run tests using this test library.
+To configure any of these, use the official documentation, to run, execute `mvn verify`.
+
 © 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
shoud it now be 2020?

Reply:
Honestly, I am not sure which year we should have there. But I don't remember updating it so maybe the year we started the project (2019) is okay.

Reply:
I think Copyright needs to be manually updated on a yearly basis. So yeah, all of the copyright notices need to be bumped up.